Annotations.md

```diff
@@ -0,0 +1,133 @@
+# Acquia RA composer.json Template w/Annotations
+
+
+
+```
+{
+    "name": "acquia/acquia-ra-composer",
+    "description": "Project json for Acquia Remote Administration Composer builds",
+```
+
+This is the name of your project.
+
+```
+    "type": "project",
+    "license": "MIT",
+```
+
+Types are used througout your composer.json. You can find more information here:
+[composer.org: Schema](https://getcomposer.org/doc/04-schema.md#type)
```

Comment:
What do you think about adding headers for each section, possibly with a table of contents?

Reply:
TOC'd up, down and sideways.